src/style-spec/expression/definitions/index.js

```diff
@@ -309,6 +309,32 @@ CompoundExpression.register(expressions, {
         varargs(NumberType),
         (ctx, args) => Math.max(...args.map(arg => arg.evaluate(ctx)))
     ],
+    'abs': [
+        NumberType,
+        [NumberType],
+        (ctx, args) => Math.abs(args[0].evaluate(ctx))
```

Comment:
Prefer argument destructuring as in the other definitions, i.e. `(ctx, [n]) => ...`.